src/main/java/com/hubspot/jinjava/lib/tag/eager/EagerImportTag.java

```diff
@@ -152,14 +157,6 @@ private static String getDoTagToPreserve(
       .getContext()
       .getSessionBindings()
       .get(currentImportAlias);
-    if ((!(currentAliasMap instanceof DeferredValue))) {
-      // Make sure that the map is deferred.
-      if (!(currentAliasMap instanceof Map)) {
-        currentAliasMap = new PyMap(new HashMap<>());
-      }
-      currentAliasMap = DeferredValue.instance(currentAliasMap);
-      interpreter.getContext().put(currentImportAlias, currentAliasMap);
-    }
```

Comment:
This is no longer needed because we are always deferring it beforehand with lines 104-108